src/enclave-agent/src/services/images.rs

```diff
@@ -173,7 +134,11 @@ mod test {
             },
         ];
 
-        let is = ImageService::new();
+        let dc = DecryptConfig::load_from_file(
+            &"test_data/decrypt_config/decrypt_config_normal.conf".to_string(),
```

Comment:
This should be the user provided `decrypt-config` param instead, right?

Reply:
Yeah, the user provides the file and specify the path the file when the agent is launched.